app/scripts/HiGlassComponent.js

```diff
@@ -2332,17 +2333,26 @@ class HiGlassComponent extends React.Component {
         // if it's a combined track, we just need to add this track to the
         // contents
         tracks.center[0].contents.push(newTrack);
+
+        if (newTrack.type === 'heatmap') {
+          this.compatibilityfyStackedHeatmaps(newTrack, tracks.center[0]);
```

Comment:
Why you doing this? Comment please.

Reply:
Oh, I see in the function definition below. A short comment would still be nice :-)

Reply:
Sure how about the comment below?
```suggestion
          // For stacked heatmaps we will adjust some options automatically for convenience
          this.compatibilityfyStackedHeatmaps(newTrack, tracks.center[0]);
```